neard/tests/rpc_nodes.rs

```diff
@@ -68,7 +79,7 @@ fn test_tx_propagation() {
                         actix::spawn(
                             client
                                 .broadcast_tx_async(to_base64(&bytes))
-                                .map_err(|err| panic!(err.to_string()))
+                                .map_err(|err| panic_on_rpc_error!(err))
```

Comment:
It looks unreasonable to ignore "IsSyncing" error given that we want to test things. We may be fooled with these tests passing while in reality, they don't even produce "Ok" result.

Reply:
why? The criterion for stopping the test didn't change.

Reply:
My bad. I got lost in these spawns.